docs/sources/flow/reference/components/prometheus.exporter.cloudwatch.md

```diff
@@ -265,12 +267,13 @@ metrics.
 Follow [this guide](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/viewing_metrics_with_cloudwatch.html)
 on how to explore metrics, to easily pick the ones you need.
 
-| Name         | Type           | Description                                                      | Default                                                   | Required |
-| ------------ | -------------- | ---------------------------------------------------------------- | --------------------------------------------------------- | -------- |
-| `name`       | `string`       | Metric name.                                                     |                                                           | yes      |
-| `statistics` | `list(string)` | List of statistics to scrape. Ex: `"Minimum"`, `"Maximum"`, etc. |                                                           | yes      |
-| `period`     | `duration`     | See [period][] section below.                                    |                                                           | yes      |
-| `length`     | `duration`     | See [period][] section below.                                    | Calculated based on `period`. See [period][] for details. | no       |
+| Name          | Type           | Description                                                        | Default                                                                                                             | Required |
+| ------------- | -------------- | ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------- | -------- |
+| `name`        | `string`       | Metric name.                                                       |                                                                                                                     | yes      |
+| `statistics`  | `list(string)` | List of statistics to scrape. Ex: `"Minimum"`, `"Maximum"`, etc.   |                                                                                                                     | yes      |
```

Comment:
```suggestion
| `statistics`  | `list(string)` | List of statistics to scrape. For example, `"Minimum"`, `"Maximum"`, etc.   |                                                                                                                     | yes      |
```

Reply:
@clayton-cornell I'm fine committing this, but I didn't actually change this text. I had to reformat because the table columns needed to get bigger to support the new row that I added.

Reply:
@berler That's fine. It's just some general cleanup we can do in the doc content - I try to catch these (the e.g. or Ex) when I spot them.